gatsby-node.js

```diff
@@ -151,6 +152,7 @@ exports.createPages = async function({ actions, graphql }) {
       slug,
       excerpt,
       brandColour,
+      seoKeywords,
```

Comment:
Just want to check on this, did we duplicate what is doen on the landing pages?

Reply:
not exactly. I did have a chat with Jason around this, and the main issue was the keywords, and rejigging what description was used, as we already have 95% of what was needed there, so rather than duplicating stuff, we only added the keywords field.